.github/workflows/cmake.yml

```
@@ -189,13 +194,15 @@ jobs:
           key: red-datasets-ubuntu
       - name: "Test: command line"
         if: env.GRN_TEST_RUN_COMMAND_LINE == 'yes'
+        continue-on-error: true
```

Can we use `true` only when we use UBSAN?

```suggestion
        continue-on-error: ${{ matrix.ubsan == 'ON' }}
```

---

fix: [6c124da](https://github.com/groonga/groonga/pull/2277/commits/6c124da31e2e807ecd5c902f0d8911f99a90a47d) Ahh I understood it. Also, I will address the same approach on Mroonga with MariaDB main CI.
ref: https://github.com/mroonga/mroonga/pull/912#issuecomment-2817614852